larray/core/array.py

```diff
@@ -1312,8 +1312,7 @@ def reindex(self, axes_to_reindex=None, new_axis=None, fill_value=np.nan, inplac
         axes_to_reindex : axis ref or dict {axis ref: axis} or list of tuple (axis ref, axis) \
                           or list of Axis or AxisCollection
             Axes to reindex. If a single axis reference is given, the `new_axis` argument must be provided.
-            If a list of Axis or an AxisCollection is given, already present axes are reindexed while missing
-            are added.
+            If a list of Axis or an AxisCollection is given, existing axes are reindexed while missing ones are added.
```

Comment:
I wonder if this is actually true. From looking at the code I suspect this is true for AxisCollection but not for a list of Axis?!

Reply:
See new commit